.gitignore

```diff
@@ -4,3 +4,36 @@
 /be/compendium/db/migrations/__pycache__
 /be/venv
 /be/.venv
+<<<<<<< Updated upstream
```

Comment:
naay wa pa na fix na conflict here @ENRexGeorge

Reply:
okay miss, on it